### PR TITLE
Document endpoints table

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,22 @@ From command-line:
 
 *Todo list here*
 
+## Design
+
+### Endpoints
```
Comment (laviro) [inline README.md:66]:
```suggestion
## Endpoints
```

so it will look like all of the other titles

**Review by laviro — COMMENTED** [review]
Nice addition! just a small nitpick

File: README.md (hunk 1 above)
Comment (ShimShtein) [reply README.md:66]: I wanted it to be a sub-title of design

**Review by laviro — APPROVED** [review]

**Comment by laviro** [comment]
Thanks @ShimShtein !

**Pull request merged**